File: domain/src/main/java/com/thinkzi/weather/domain/repository/IWeatherRepository.java
package com.thinkzi.weather.domain.repository;

import com.thinkzi.weather.domain.entity.WeatherInCity;
import io.reactivex.Single;

public interface IWeatherRepository {

    Single<WeatherInCity> getWeatherForFiveDaysInACity(int _cityId, String _appId);

}

File: domain/src/main/java/com/thinkzi/weather/domain/repository/IWeatherRepository.kt
package com.thinkzi.weather.domain.repository

import com.thinkzi.weather.domain.entity.WeatherInCity
import io.reactivex.Single

interface IWeatherRepository {

    fun getWeatherForFiveDaysInACity(_cityId: Int, _appId: String): Single<WeatherInCity>

}
